src/reducers/updateScrollableColumn.js

```diff
@@ -54,6 +54,7 @@ export function getScrollableColumn(state, localColIdx) {
       _.get(storedColumnGroup, 'props.lastChildIdx'),
     ]);
 
+    columnGroup.props.width = _.get(storedColumnGroup, 'props.width');
```

Comment:
Don't we need to update the column group's width based on the current scrollable column's width?

Reply:
No, not here.  `computeRenderedColumnGroups` will take care of storing correct width in `storedColumnGroup`

Reply:
I'm confused. I thought the bug here is specific to the scenario where `computeRenderedColumnGroups()` doesn't get called for whatever reason.

Reply:
`computeRenderedColumnGroups()` is getting called when it should be,   The bug here is that this `updateScrollableColumn` get called sometimes after `computeRenderedColumnGroups()` for whatever reason and set the width in `storedColumnGroup` to undefined.

Reply:
I get that this makes `width` not be reset to `undefined`.
But my concern is over whether column group width needs to be updated based on the new scrollable column.

If the order of calls are something like:
```
updateScrollableColumn()
computeRenderedColumnGroups()
updateScrollableColumn()
```
Then in the second call to `updateScrollableColumn()`, the column being fetched can have a different width, but the column group associated with the column still has the same width.

I don't know how much of a deal this'll be, but to figure this out, we'll need to know more about why/how the second call to updateScrollableColumn() happens.
If it gets called on visible columns, then we'll probably need to update the column group width. Otherwise, it _may_ be safe to ignore.

Reply:
I don't know.  what i observed is there are some cases where nothing change in props but propsChange action get fired, and some of these case lead to call `updateScrollableColumn()` without `computeRenderedColumnGroups()`.   for all other cases both of them get called and hence width are usually correct.

We can file separate bug to investigate about those unnecessary calls to updateScrollableColumns.

At that point, i think this is just a quick and good workaround to fix that bug in LD. we could invest more time in debugging that if either this solution won't work in some edge case or there is some other driver.

Reply:
Offlined. The calls to `updateScrollableColumn()` most likely come from https://github.com/schrodinger/fixed-data-table-2/blob/e45a2fc05ffaf66c8442e1d36cc11fcf8a1987fc/src/reducers/computeRenderedCols.js#L482-L497

The columns being fetched anew here are outside the viewport, and come from the previous render's virtualized column set,
So it should be safe to ignore updating the corresponding column group width.